src/istio/control/http/attributes_builder_test.cc

```diff
@@ -14,6 +14,7 @@
  */
 
 #include "src/istio/control/http/attributes_builder.h"
+#include "google/protobuf/stubs/status.h"
```

Comment:
nit: I believe we have a tool to reorder the header files

Reply:
Would love to learn more about this tool.  Until then, I manually fixed this on #2117